scripts/image-builder/config/kickstart.ks.template

```diff
@@ -99,4 +99,11 @@ if ostree remote show-url edge | grep -q '^file:///var/lib/ostree-local' ; then
     ostree summary --repo ${ostree_repo}/repo --update
 fi
 
-%end
+# Disable a timer that sets boot_success to 1 after two minutes from a user login.
+# It impacts greenboot checks because grub script decrementing boot_counter works
+# only if boot_success is 0 (see /etc/grub.d/08_fallback_counting). In case of a
+# user login, this results in more than requested amount of red-boot-induced reboots
+# and system needing much more time to roll back.
+systemctl mask grub-boot-success.timer
```

Comment:
This doesn't seem to work. Although the timer is masked, it's not masked for a user
```
# cat /var/log/anaconda/post-install.log 
Created symlink /etc/systemd/system/grub-boot-success.timer -> /dev/null.
```

When I log in, it's active:
```
$ systemctl --user status grub-boot-success.timer
● grub-boot-success.timer - Mark boot as successful after the user session has run 2 minutes
     Loaded: loaded (/usr/lib/systemd/user/grub-boot-success.timer; static)
     Active: active (elapsed) since Tue 2023-07-25 13:00:22 EDT; 2min 57s ago
      Until: Tue 2023-07-25 13:00:22 EDT; 2min 57s ago
    Trigger: n/a
   Triggers: ● grub-boot-success.service

Jul 25 13:00:22 rhel-9-2-microshift-source-block-upgrade-2-minor-host1 systemd[5782]: Started Mark boot as successful after the user session has run 2 minutes.

$ systemctl --user status grub-boot-success.service
○ grub-boot-success.service - Mark boot as successful
     Loaded: loaded (/usr/lib/systemd/user/grub-boot-success.service; static)
     Active: inactive (dead) since Tue 2023-07-25 13:02:36 EDT; 1min 36s ago
TriggeredBy: ● grub-boot-success.timer
    Process: 6958 ExecStart=/usr/sbin/grub2-set-bootflag boot_success (code=exited, status=0/SUCCESS)
   Main PID: 6958 (code=exited, status=0/SUCCESS)
        CPU: 1ms

Jul 25 13:02:36 rhel-9-2-microshift-source-block-upgrade-2-minor-host1 systemd[5782]: Starting Mark boot as successful...
Jul 25 13:02:36 rhel-9-2-microshift-source-block-upgrade-2-minor-host1 systemd[5782]: Finished Mark boot as successful.
```

Note that the command in the test is: `systemctl --user mask`